scripts/lint.sh

```diff
@@ -4,4 +4,4 @@ set -e # exit when error
 
 printf "\nLint\n"
 
-eslint . --quiet
+eslint . --quiet --no-color
```

Comment:
Shouldn't have pushed this file, will remove it. But was because of `solarized dark` that doesn't work well with the output of `eslint`.